engine/src/flutter/lib/web_ui/lib/src/engine/semantics/checkable.dart

```diff
@@ -40,6 +40,20 @@ _CheckableKind _checkableKindFromSemanticsFlag(SemanticsObject semanticsObject)
   }
 }
 
+class SemanticRadioGroup extends SemanticRole {
```

Comment:
Can you add some document to this class?

Reply:
Added!